composer.json

```diff
@@ -19,39 +19,41 @@
         "erusev/parsedown": "^1.6",
         "sensio/framework-extra-bundle": "^6.1",
         "symfony/apache-pack": "^1.0",
-        "symfony/asset": "^5.3",
-        "symfony/console": "^5.3",
-        "symfony/dotenv": "^5.3",
-        "symfony/expression-language": "^5.3",
+        "symfony/asset": "5.4.0-beta1",
+        "symfony/console": "5.4.0-beta1",
+        "symfony/dotenv": "5.4.0-beta1",
+        "symfony/expression-language": "5.4.0-beta1",
         "symfony/flex": "^1.1",
```

Comment:
should probably be updated to `^1.17.1` due to https://symfony.com/blog/upgrade-flex-on-your-symfony-projects

Reply:
Yes, this is very important. Thanks!